nexus/tests/integration_tests/unauthorized_coverage.rs

```diff
@@ -126,11 +126,14 @@ fn test_unauthorized_coverage() {
     // If you're here because this assertion failed, check that if you've added
     // any API operations to Nexus, you've also added a corresponding test in
     // "unauthorized.rs" so that it will automatically be checked for its
-    // behavior for unauthenticated and unauthorized users.
-    assert_contents(
-        "tests/output/uncovered-authz-endpoints.txt",
-        &uncovered_endpoints,
-    );
+    // behavior for unauthenticated and unauthorized users.  DO NOT SKIP THIS.
+    // Even if you're just adding a stub, see [`Nexus::unimplemented_todo()`].
+    // If you _added_ a test that covered an endpoint from the allowlist --
+    // hooray!  Just delete the corresponding line from this file.
```

Comment:
yes! You might even say:

Why is this not `expectorate:::assert_contents`? Because this file should only shrink a line at a time; if you find yourself adding something you're probably doing something wrong and you deserve to suffer. Muahahahaha.

Reply:
Added a comment about this in c4d8e1fbd217a323522893f218a191f47c375f71.  Thanks!

Reply:
Less colorful than I would have preferred... but ok.